app/screens/select_server/select_server.js

```diff
@@ -206,6 +206,19 @@ export default class SelectServer extends PureComponent {
             return;
         }
 
+        if (!this.state.url || this.state.url.trim() === '') {
```

Comment:
I think the conditional can also be written this way:

` if (!this.state.url?.trim()) `

I prefer the version in the PR, but I'm 1/5. Either way is fine with me.

Reply:
the ways is done in the PR is fine, but I like the suggestion ;)